misc/multiaddr/src/lib.rs

```diff
@@ -412,3 +423,26 @@ macro_rules! multiaddr {
         }
     }
 }
+
+struct Buffer(smallvec::SmallVec<[u8; 32]>);
```

Comment:
`smallvec` has a `write` feature that would allow you to remove `Buffer`.

Reply:
:+1: